LeaderboardBackend.Test/Fixtures/PostgresDatabaseFixture.cs

```diff
@@ -43,61 +42,4 @@ public static async Task OneTimeTearDown()
 
         await PostgresContainer.DisposeAsync();
     }
-
-    public static void CreateTemplateFromCurrentDb()
```

Comment:
Why this significant change to how we reset the DB?

Reply:
Because it wasn't working with EF 9 and it was kind of hacky anyway. It's way easier to just delete everything.